src/actors/hcd/neutral_fueling_actor.jl

```diff
@@ -0,0 +1,417 @@
+#= ====== =#
+#  PELLET  #
+#= ====== =#
+Base.@kwdef mutable struct FUSEparameters__ActorNeutralFueling{T<:Real} <: ParametersActor{T}
+    _parent::WeakRef = WeakRef(nothing)
+    _name::Symbol = :not_set
+    _time::Float64 = NaN
+    taupin::Entry{Float64} = Entry{Float64}("-", "Particle confinement time"; default=1.0)
+    twall::Entry{Float64} = Entry{Float64}("-", "Wall temperature (eV)"; default=10.0)
+    vwall::Entry{Float64} = Entry{Float64}("-", "Wall particle velocity (m/s)"; default=100.0)
```

Comment:
```suggestion
    taupin::Entry{Float64} = Entry{Float64}("s", "Particle confinement time"; default=1.0)
    twall::Entry{Float64} = Entry{Float64}("eV", "Wall temperature"; default=10.0)
    vwall::Entry{Float64} = Entry{Float64}("m/s", "Wall particle velocity"; default=100.0)
```

Reply:
I would also be inclined not to assign a default value to taupin

Other thoughts:
`taupin` --> `τp`
`twall` --> `T_wall`
`vwall` --> `v_wall`